### ISSUE TITLE
Visual Bug in API reference section

### ISSUE THREAD

**Issue by rapax00** [lacrypta/wallet-docs#3]
# Bug
While reading the documentation, I noticed that the `/fetch` endpoint has a **GET** badge, even though it uses a **POST** method.

Watch it:
![image](https://github.com/user-attachments/assets/baea44dd-91e7-4c4b-bad0-c59c5966df29)

# Solution?

I think its a good idea, as good neighbor, to send a PR. I cloneed and run this repo locally, but surprise! Locally, it's correct, although other endpoints don't have the badge.

Watch it:
<table>
  <tr>
    <td>Local</td>
    <td>Production</td>
  </tr>
  <tr>
    <td>
      <ul>
        <li>Badge of <code>/fetch</code> is correct</li>
        <li>Not shown badges of other endpoints</li>
      </ul>
    </td>
    <td>
      <ul>
        <li>Badge of <code>/fetch</code> is wrong</li>
        <li>Shown badges of other endpoints</li>
      </ul>
    </td>
  </tr>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/25084a2e-9a73-4e12-a8d5-e0d30d0cc062" alt="local"
        width="300" /></td>
    <td><img src="https://github.com/user-attachments/assets/6b3b6fad-a6c5-4642-a8b2-b6ba0eb6281d" alt="production"
        width="300" /></td>
  </tr>
</table>

# In summary

I'm not sure what's happening, whether it's a problem with how the repo is being deployed (maybe an old commit) or something at the code level. 

I hope for a response to be able to collaborate and send a PR.
